tests/api/v2/datadog/security_monitoring_test.go

```diff
@@ -0,0 +1,180 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/DataDog/datadog-api-client-go/tests"
+	_nethttp "net/http"
+	"testing"
+)
+
+func createRule(ctx context.Context, api *datadog.SecurityMonitoringApiService, ruleName string) (datadog.SecurityMonitoringRuleResponse, *_nethttp.Response, error) {
+
+	query := datadog.NewSecurityMonitoringRuleQuery()
+	query.SetName("nevermatch")
+	query.SetQuery("thiswillnevernevermatch")
+	query.SetGroupByFields([]string{})
+	queries := []datadog.SecurityMonitoringRuleQuery{*query}
+
+	options := datadog.NewSecurityMonitoringRuleOptions()
+	options.SetEvaluationWindow(datadog.SECURITYMONITORINGRULEEVALUATIONWINDOW_FIVE_MINUTES)
+	options.SetKeepAlive(datadog.SECURITYMONITORINGRULEKEEPALIVE_FIVE_MINUTES)
+	options.SetMaxSignalDuration(datadog.SECURITYMONITORINGRULEMAXSIGNALDURATION_FIVE_MINUTES)
+
+	ruleCase := datadog.NewSecurityMonitoringRuleCase()
+	ruleCase.SetName("rule-case")
+	ruleCase.SetCondition("nevermatch > 1000")
+	ruleCase.SetStatus(datadog.SECURITYMONITORINGRULESEVERITY_INFO)
+	ruleCases := []datadog.SecurityMonitoringRuleCase{*ruleCase}
+
+	createPayload := datadog.NewSecurityMonitoringRuleCreatePayload(
+		ruleCases,
+		true,
+		"rule message",
+		ruleName,
+		*options,
+		queries,
+		[]string{"datadog-api-client-test-go"},
+		)
+
+	return api.CreateSecurityMonitoringRule(ctx).Body(*createPayload).Execute()
+}
+
+func deleteRule(t *testing.T, ctx context.Context, api *datadog.SecurityMonitoringApiService, id string) {
+	_, err := api.DeleteSecurityMonitoringRule(ctx, id).Execute()
+	if err != nil {
+		t.Logf("Error delete rule: Response %s: %v", err.(datadog.GenericOpenAPIError).Body(), err)
+	}
+}
+
+func TestSecMonRulesCRUD(t *testing.T) {
+	ctx, finish := WithRecorder(WithTestAuth(context.Background()), t)
+	defer finish()
+	assert := tests.Assert(ctx, t)
+	api := Client(ctx).SecurityMonitoringApi
+
+	ruleResponses := []datadog.SecurityMonitoringRuleResponse{}
+	now := tests.ClockFromContext(ctx).Now().UnixNano()
+	// create rules
+	for i := 0; i < 5; i++ {
+		ruleName := fmt.Sprintf("test-rule-datadog-api-client-go-%d-%d", i, now)
```

Comment:
Can you use [`UniqueEntityName`](https://github.com/DataDog/datadog-api-client-go/blob/master/tests/test_utils.go#L179) from `tests/test_utils.go`?